public/logo-albert.svg

```diff
@@ -0,0 +1,41 @@
+  </g>
+  <g id="Gruppe_14" data-name="Gruppe 14" transform="translate(62.733 51.881)">

```

Comment:
Some of these properties are export artifacts. You can remove them (id, data-name).
It's possible to automatize this by using SVG cleaner/minifier tools.